src/components/homepage/featureSection.tsx

```diff
@@ -0,0 +1,109 @@
+import { useRef } from 'react';
+import Image from 'next/image';
+import styles from '@/pages/mainpage.module.css';
+import Information from '@/public/images/serverInformation.png';
+import TeamChat from '@/public/images/teamchat.png';
+import EventNotifications from '@/public/images/eventNotifications.png';
+import SmartSwitch from '@/public/images/smartSwitch.png';
+import Activity from '@/public/images/activity.png';
+import Trackers from '@/public/images/trackersChannel.png';
+import useIntersectionObserver from '@/hooks/useIntersectionObserver';
+
+const FeatureSection = () => {
+  const sectionRefs = [
+    useRef<HTMLElement>(null),
+    useRef<HTMLElement>(null),
+    useRef<HTMLElement>(null),
+    useRef<HTMLElement>(null),
+    useRef<HTMLElement>(null),
+    useRef<HTMLElement>(null),
+  ];
+
+  useIntersectionObserver(
+    sectionRefs,
+    (entry) => {
+      if (entry.isIntersecting) {
+        entry.target.classList.add(styles.visible);
+      }
+    },
+    { threshold: 0.7 },
+  );
+
+  return (
+    <>
+      <section ref={sectionRefs[0]} className={styles.section}>
+        <div className={styles.sectionImg}>
+          <Image src={Information} alt="server information" />
+        </div>
+        <div className={styles.sectionText}>
+          <h2>Information</h2>
+          <p>
+            Check the map, players, in-game time, event information, and team member details and status of the currently
+            connected server in Discord!
+          </p>
+        </div>
+      </section>
```

Comment:
Creating a new component and using the map function for this part will improve the readability and maintainability of the code.